documentation/asciidoc/computers/remote-access/find-your-ip-address.adoc

```diff
@@ -110,35 +116,41 @@ PING raspberrypi.local (192.168.1.131): 56 data bytes
 
 If you change the system hostname of the Raspberry Pi (e.g., by editing `/etc/hostname`), Avahi also changes the `.local` mDNS address.
```

Comment:
I suspect these days we'd now recommend people to use raspi-config or Raspberry Pi Configuration to change their hostname, rather than directly editing `/etc/hostname`.